### ISSUE TITLE
Inconsistence between the pipeline strategy and its wizard screens

### ISSUE THREAD

**Issue by rafabene** [openshiftio/openshift.io#1028]
During the Quickstart project wizard, the Pipeline strategy mentions and shows "Integration tests"

![screenshot 2017-09-26 16 14 25](https://user-images.githubusercontent.com/30131/30882047-ccb74412-a2d5-11e7-9073-be5539b207f8.png)

However the "Integration tests" is not present anymore in the default stragegy

![screenshot 2017-09-26 16 15 03](https://user-images.githubusercontent.com/30131/30882070-e6ddb222-a2d5-11e7-9758-519e9bed4f1d.png)

**Comment by qodfathr** [comment]
I don't care where this gets fixed -- either in the App Wizard or in the pipeline definitions.

**Comment by qodfathr** [comment]
Will likely be addressed by new appwizard design.

**Comment by kishansagathiya** [comment]
This is UI/UX related and not build related. Build team at the moment doesnt take care of UI.
@joshuawilson @pradeepto We should remove `team/build-cd` tag from this

**Comment by chmouel** [comment]
@kishansagathiya let's be clear, build team cares about Ui it just we don't have resources for it at the moment, we should not remove this tag and find a way if we can find someone who can help us on this instead,

**Comment by pradeepto** [comment]
Agreed with @chmouel I will work on finding someone who can help us with that.

**Comment by joshuawilson** [comment]
@chmouel Can you confirm that there will be no "integration test" stage? If so, then this is an issue for the Launcher/Wizard team. 

@qodfathr As the current app-wizard is deprecated and not being worked on, I'm going to reduce this to a SEV3 and remove the P0. If you disagree, please reset it.

**Comment by rupalibehera** [comment]
If we do not run any integration tests and this is the issue with the launcher team to look in to , as we need to remove the "Integration Tests" from the wizard.

**Comment by rupalibehera** [comment]
> @chmouel Can you confirm that there will be no "integration test" stage? If so, then this is an issue for the Launcher/Wizard team.
@joshuawilson , there is no stage for integration tests.
> 
> @qodfathr As the current app-wizard is deprecated and not being worked on, I'm going to reduce this to a SEV3 and remove the P0. If you disagree, please reset it.

**Comment by joshuawilson** [comment]
Then this is just an app-wizard issue.

**Comment by invincibleJai** [comment]
@joshuawilson so if I am understanding it correctly the expectation here not to show integration step in pipelines in the launcher irrespective of which booster/import user selected? 

cc @animuk

**Comment by joshuawilson** [comment]
@invincibleJai it looks like Build does not support an Integration Test phase/stage. Therefore, it should not be listed on the page in the wizard describing the pipelines.

**Comment by invincibleJai** [comment]
@animuk @ravsa IMO It needs to be removed from API response as well if it's not needed.

`https://*****/api/services/jenkins/pipelines` returns stages list as well along with description for each stage 

eg: 

```
[{name: "Build Release", description: "creates a new version then builds and deploys the project into the maven repository"}
{name: "Integration Test", description: "runs an integration test in the Test environment"}]
```

am not sure if this will OSIO only change or needed for launcher-frontend as well. CC @gastaldi

**Comment by gastaldi** [comment]
@invincibleJai launcher-frontend doesn't use `/api/services/jenkins/pipelines`, so feel free to change the data in https://github.com/fabric8io/fabric8-jenkinsfile-library however it fits
